### PR TITLE
PowerShell Support for Email Services Control Plane CRUD operations

### PR DESCRIPTION
**Pull request by v-vprasannak**
## Version 0.1.0
* First preview release for module Az.EmailService

## Mandatory Checklist
[ChangeLog.md](https://github.com/Azure/azure-powershell/files/14384825/ChangeLog.md)

- Please choose the target release of Azure PowerShell
  - [x] [General release](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Public preview](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Private preview](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Engineering build](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] N/A

- [ ] Check this box to confirm: **I have read the [_Submitting Changes_](../blob/main/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/main/CONTRIBUTING.md) and reviewed the following information:**

* **SHOULD** update `ChangeLog.md` file(s) appropriately
    * For SDK-based development mode, update `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`.
        * A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header in the past tense. 
    * For autorest-based development mode, include the changelog in the PR description.
    * Should **not** change `ChangeLog.md` if no new release is required, such as fixing test case only.
* **SHOULD** regenerate markdown help files if there is cmdlet API change. [Instruction](../blob/main/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)
* **SHOULD** have proper test coverage for changes in pull request.
* **SHOULD NOT** adjust version of module manually in pull request

### CODE REVIEW

**Review by msJinLei — CHANGES_REQUESTED** [review]
The folder name should be consistent with module name.
Please change folder name to emailservice or module name to communication

**Comment by v-vprasannak** [comment]
> The folder name should be consistent with module name. Please change folder name to emailservice or module name to communication
@msJinLei Could you please suggest, is there any particular pattern to be followed? or can we go ahead with the below pattern?
- src
     - Communication
            - Communication.Autorest
     - EmailService
           - EmailService.Autorest

**Comment by msJinLei** [comment]
> > The folder name should be consistent with module name. Please change folder name to emailservice or module name to communication
> > @msJinLei Could you please suggest, is there any particular pattern to be followed? or can we go ahead with the below pattern?
> 
> * src
>   
>   * Communication
>     - Communication.Autorest
>   * EmailService
>     - EmailService.Autorest

Please add 
```
root-module-name: $(prefix).Communication
``
to the readme.md of EmailService

File: src/Communication/EmailService.Autorest/README.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+# Normally, title is the service name
+title: EmailService
+subject-prefix: $(service-name)
+
+# If there are post APIs for some kinds of actions in the RP, you may need to 
+# uncomment following line to support viaIdentity for these post APIs
+# identity-correction-for-post: true
+
+# For new modules, please avoid setting 3.x using the use-extension method and instead, use 4.x as the default option
+use-extension:
```
Comment (msJinLei) [inline src/Communication/EmailService.Autorest/README.md]:
```suggestion
```

File: src/Communication/EmailService.Autorest/README.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+  "@autorest/powershell": "4.x"
```
Comment (msJinLei) [inline src/Communication/EmailService.Autorest/README.md]:
```suggestion
```

**Review by msJinLei — APPROVED** [review]

**Review by msJinLei — APPROVED** [review]

**Pull request merged**